mysql/datadog_checks/mysql/innodb_metrics.py

```diff
@@ -8,7 +8,7 @@
 import pymysql
```

Comment:
In which case do we receive bytes ? Is there a way to tell the library to return native string only ?